source/apiVolontaria/apiVolontaria/tests/tests_view_Users.py

```diff
@@ -303,21 +343,70 @@ def test_create_user_activation_email_failure(self, imailing):
             'last_name': 'Norris',
         }
 
-        instance_imailing = imailing.create_instance.return_value
-        instance_imailing.send_templated_email.return_value = {
-            "code": "failure",
+        response = self.client.post(
+            reverse('users'),
+            data,
+            format='json',
+        )
+
+        # Test that one message was sent:
+        self.assertEqual(len(mail.outbox), 0)
+
+        content = {
+            'detail': "The email service is not activated.",
```

Comment:
This message is not really clear for the customer. Maybe can you try this :

```
The account was created but no email was sent (email service deactivated). If your account is not activated, contact the administration.
```

Reply:
done